docs/setting-up-your-environment.md

```diff
@@ -30,6 +30,25 @@ On recent versions of Ubuntu and Debian, you can run the following
 command to ensure you have the required libraries, headers, and tools:
 
     sudo apt-get install build-essential git libicu-dev zlib1g-dev
+    
+###### Mac OSX differences:
```

Comment:
You can just have this as "Mac OSX Systems" and add a header above titled "Ubuntu/Debian Systems"

Reply:
I'll make the change.